docs/usage/gitlab.html.md

```diff
@@ -11,6 +11,10 @@ variable on your CI system:
 
 - `DANGER_GITLAB_API_TOKEN` = An access token for the account which will post comments
 
+You will need also to add an environment variable for your gitlab project id:
```

Comment:
```suggestion
You may need also to add an environment variable for your Gitlab project id:
```

Reply:
If we get this in, I'm good 👍

Reply:
@orta Can you edit `Gitlab` to `GitLab` in the suggestion!